packages/example/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import { Layout } from 'antd';
+import './App.css';
+import { Outlet } from 'react-router-dom';
+import { NavigationMenu } from './components/NavigationMenu';
+
+const { Header, Content, Footer } = Layout;
+
+export function App(): JSX.Element {
+  return (
+    <Layout className="layout">
+      <Header className="header" style={{ height: 67 }}>
+        <div className="logo" />
+        <NavigationMenu />
+      </Header>
+      <Content style={{ padding: '50px 50px', minHeight: 'calc(100vh - 67px - 64px)' }}>
```

Comment:
'calc(100vh - 67px - 64px)'
67 and 64 it's probably a Header and Footer height, can we make it a constants to not create a confusion from where this numbers comes from